release/hub.conf

```diff
@@ -1,3 +1,3 @@
-HUB_API_URL=http://localhost/rpc/api
+HUB_API_URL=https://localhost/rpc/api
```

Comment:
Does `http` still work fine?

Does `https` require any extra configuration in case the daemon is installed on the Hub? What if it is installed on another machine?

Reply:
> Does `http` still work fine?
> 

Yes, HTTP still works fine.

> Does `https` require any extra configuration in case the daemon is installed on the Hub? What if it is installed on another machine?

To make https works, whichever machine we use Hub or another machine---that machine needs to have all the certificates of prephiral servers.

Reply:
> > Does `http` still work fine?
> 
> Yes, HTTP still works fine.

Then you might want to keep http and a deployment on Hub itself as the base case for examples, as they are the simplest situation.

> > Does `https` require any extra configuration in case the daemon is installed on the Hub? What if it is installed on another machine?
> 
> To make https works, whichever machine we use Hub or another machine---that machine needs to have all the certificates of prephiral servers.

Then please add a paragraph on the project README and corresponding documentation in Uyuni on how those should be added.

Thanks!